examples/src/main/java/com/hedera/hashgraph/sdk/examples/advanced/DeleteFile.java

```diff
@@ -64,6 +63,7 @@ public static void main(String[] args) throws HederaException {
 
         FileInfo fileInfo = new FileInfoQuery()
             .setFileId(newFileId)
+            .setPaymentAmount(Hbar.of(2))
```

Comment:
Why are you setting a explicit payment amount here? Is there a bug in the network? I ask because you **should** be able to set a _maximum_ with `.setMaxPaymentAmount(Hbar.of(2))` and then the SDK will dynamically ask the cost of the query and pay the exact amount as long as its under your configured maximum (the default being 1h).

Reply:
I had an older sdk from a month ago so I just grabbed the latest one right now to check
 and I have options for setMaxQueryPayment, setQueryPayment, and depreciated setPaymentAmount. If I use setMaxQueryPayment it fails and returns an insufficient transaction fee response. If I use setQueryPayment or setPaymentAmount it is successful. Screenshots:

![Screen Shot 2020-01-07 at 12 15 37 AM](https://user-images.githubusercontent.com/21989138/71879492-0e122100-30e3-11ea-9146-26fa349b9f9a.png)
![Screen Shot 2020-01-07 at 12 16 07 AM](https://user-images.githubusercontent.com/21989138/71879506-123e3e80-30e3-11ea-85d6-1d074babe3b3.png)

Reply:
That feels like a bug in the network and the response from the COST_ANSWER query. I'll open an issue to investigate this further. Do leave in the explicit payment amount for now.

Reply:
The network returns a `COST_ANSWER` of zero because the file is deleted.